omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -100,61 +100,65 @@ Requirements
 
    Homebrew installs Python in the following location::
 
-    '/usr/local/opt/python/libexec/bin'
+    '/usr/local/opt/python3/libexec/bin'
 
    Follow the instructions from the brew Python install and set your system to use the Homebrew version of Python 
    rather than the Python shipped with OS X. Add the following line to your :file:`.bash_profile`::
 
-    export PATH="/usr/local/opt/python/libexec/bin:$PATH"
+    export PATH="/usr/local/opt/python3/libexec/bin:$PATH"
 
-5. Check that Python is working and is version 2.7.x::
+5. Check that Python is working and is version 3.7.x::
 
-    $ which python
-    /usr/local/opt/python/libexec/bin/python
+    $ which python3
+    /usr/local/opt/python3/libexec/bin/python
 
     $ python --version
-    Python 2.7.13
+    Python 3.7.4
 
 6. For developing with OMERO, or Python in general, we recommend the use of Virtualenv.
    Virtualenv allows development of Python applications without having to
    worry about clashing third-party packages for different Python projects.
 
-   Use pip to get `Virtualenv <https://virtualenv.pypa.io/en/stable/>`__::
+   Create a virtual environment for omero-py using Python 3::
 
-    $ pip install virtualenv
+    $ python3 -mvenv ~/Virtual/omero-server
 
-   With Virtualenv installed, create a virtual environment::
+   This will create a folder to hold Python libraries in the the directory :file:`~/Virtual/omero-server/lib`
 
-    $ virtualenv ~/Virtual/omero
+   Create a virtual environment for omero-web using Python 3::
 
-   This will create a folder to hold Python libraries in the the directory :file:`~/Virtual/omero/lib`
+    $ python3 -mvenv ~/Virtual/omero-web
+
+   This will create a folder to hold Python libraries in the the directory :file:`~/Virtual/omero-web/lib`
 
   .. note:: 
    You can activate the Virtualenv environment that we created using::
 
-    $ source ~/Virtual/omero/bin/activate
+    $ source ~/Virtual/omero-server/bin/activate
 
    This will switch to using Pip and Python in the Virtualenv directory
-   :file:`~/Virtual/omero/bin` and any Pip libraries you install, whilst the Virtualenv is activated, 
-   will be installed to :file:`source ~/Virtual/omero/lib`.
+   :file:`~/Virtual/omero-server/bin` and any Pip libraries you install, whilst the Virtualenv is activated, 
+   will be installed to :file:`source ~/Virtual/omero-server/lib`.
 
   .. note::
    **(Optional)** To make starting a Virtualenv environment easier,
    you can add an `alias` to your :file:`.bash_profile`::
 
-    alias startVmOmero="source ~/Virtual/omero/bin/activate"
+    alias startVmOmeroServer="source ~/Virtual/omero-server/bin/activate"
+    alias startVmOmeroWeb="source ~/Virtual/omero-web/bin/activate"
 
    Using the command-line terminal, reload your :file:`.bash_profile`::
 
     $ source ~/.bash_profile
 
    Now you can activate the Virtualenv environment using::
 
-    $ startVmOmero
+    $ startVmOmeroServer
+    $ startVmOmeroWeb
```

Comment:
It might be nice to use these commands instead of the longer ```source...``` commands everywhere they're used below (or add the alias alongside the source command) so that by the time you are done, you've learnt them. Maybe the "(Optional)" should be "(Recommended)". I've certainly found it useful to have these aliases.

Reply:
My preference would be to _not_ have the aliases _anywhere_. If the path to the directory is too long/difficult to remember, I'd suggest we find a location that is more user friendly. `source` (and `.` even more so) is a fundamental shell action (like `ls`, `cd`, and `mv`). Depending on the alias means that a user in another situation may not have learned "source" which would ultimately be more useful.

Reply:
the aliases have been there for a while. Happy to remove them 
Is it a blocker?

Reply:
Not a blocker (unless someone is relying on them in production?) and not a must from my side. Just spelling out my preference how I think about the doc design.